crates/jstzd/src/task/octez_client.rs

```diff
@@ -133,6 +135,11 @@ impl OctezClient {
         let mut command = self.command(args)?;
         Ok(command.spawn()?)
     }
+
+    pub async fn config_init(&self, output_path: &Path) -> Result<Child> {
+        let output = output_path.to_str().ok_or(anyhow!("non utf-8 path"))?;
```

Comment:
Maybe make this error message more specific? Something like `config output path must be a valid utf-8 path`.